nevergrad/benchmark/experiments.py

```diff
@@ -42,7 +42,9 @@ def mltuning(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     # First, a few functions with constraints.
     functions = [
         MLTuning("1d_decision_tree_regression"),
-        MLTuning("1d_decision_tree_regression_full")
+        MLTuning("1d_decision_tree_regression_full"),
+        MLTuning("2d_decision_tree_regression_full"),
+        MLTuning("3d_decision_tree_regression_full"),
```

Comment:
instead of having so many different names, maybe you would need "dimension" and "full" parameters instead?
And for test cases which are very different, creating a different class may be simpler
I'm a bit afraid there starts to be a lot of `if` conditions everywhere, this is never a good sign

Reply:
I'm not sure I agree... we will have other test cases, including sophisticated cases in which Nevergrad will choose between various MLP and Decision trees and others. So we will need a function with everything.

Reply:
Thanks for the offline comments, which help a lot :-)   I update accordingly.